README.md

```diff
@@ -87,13 +87,20 @@ out" (using upper-case variable names is a common way to do this).
 
 ### Comments in code
 
-Comments should explain *why* the code does what it does. *What* it
-does should ideally already be evident from the code itself. If the code
-is cryptic and can't easily be simplified, explanations might well be
-needed.
+Comments should explain *What* the code does, and *why* the code does
+what it does. *What* it does should ideally also be evident from the
+code itself, but a short description should be included to allow a new
+developer to get an overview of the code quicker.
```

Comment:
Hmm... Comments are not written just to get new developers up to speed but also to justify the code (another way to say "clarify intent"). I would drop the part of the sentence after "should be included".